config/site_settings.yml

```diff
@@ -408,7 +408,6 @@ users:
     default: true
   invite_expiry_days:
     default: 30
-  invite_passthrough_hours: 0
```

Comment:
We need to write a migration to drop site settings in the db here

Reply:
Done.